wallet/how-to/connect/set-up-sdk/index.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 1
+---
+
```

Comment:
Are we able to make this autogenerate the cards / items below with a simlar file as [wallet/how-to/connect/_category_.json](https://github.com/MetaMask/metamask-docs/pull/840/files#diff-b404e0f9ae143b54c920baf72661855ec09c300f25b578ba997de6ba004eec25) ? Not sure, but I'd imagine that the `description` part of the _category_.json would support markdown. The only thing that would be missing would be the note at the bottom I think?

reference: https://docusaurus.io/docs/sidebar/autogenerated#autogenerated-sidebar-metadata

Reply:
The autogenerated category cards by default don't support displaying bulleted lists with different links, which is the primary need for this page.

I slightly modified the style of the SDK cards to better match the category cards. Let me know if there's a better solution!

Reply:
Swizzling those components to achieve what we currently have would probably take me a long time to figure out, as I'm not experienced with React. I don't think any of the other category pages would benefit from such a feature currently. Could we stick with my workaround for this PR and return to this issue in the future? Or if someone else is willing to experiment with that, or let me know of the specific solution, that's fine too.